cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -390,3 +396,66 @@ def test_change_network_security_groups_per_page_items():
         1524443
     """
     pass
+
+
+@test_requirements.relationships
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.meta(coverage=[1729953])
+def test_datastore_relationships():
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Cloud
+        caseimportance: medium
+        initialEstimate: 1/12h
+        tags: relationship
+        setup:
+            1. Add infrastructure provider(e.g: vsphere65)
```

Comment:
Don't you also need to setup for SSA?

Reply:
Yes @john-dupuy 
- I discussed with @sbulage and I will take care of SSA while automating this PR. So I added setup step for SSA.
- Also this can be covered under any of the smartstate analysis testcase.

What are your views on this @sbulage ?

Reply:
@ganeshhubale We will sit together and I will update SSA part there. In my opinion, you should consider one test which will accommodate SSA functionality as well. Please let me know your concerns on it.

Reply:
@ganeshhubale For this PR, you can just add `2. setup SSA`

Reply:
yeah, actually we are going to perform smartstate analysis in that test case too.
Thanks :)